decidim-initiatives/app/views/decidim/initiatives/initiatives/print.html.erb

```diff
@@ -1,156 +1,158 @@
-<div class="initiative-letterhead">
-  <% if current_organization.logo.present? %>
-    <%= image_tag current_organization.logo.url, title: current_organization.name %>
-  <% end %>
-
-  <%= current_organization.name %>
-</div>
-<h3 class="print-section-title"><%= t ".section" %></h3>
-<div class="initiative-print-title">
-  <%= t ".general_title" %>
-</div>
-
-<div class="print-table">
-  <div class="print-table-header">
-    <%= t ".author_title" %>
-  </div>
+<div class="wrapper">
```

Comment:
It seems like I've changed all the file, but really there are a couple lines changed. The thing is that as I've added this `.wrapper` I've also changed the indentation and that's bad for git diff. Maybe it's better to review on a commit by commit basis. 

I could also cherry-pick all the different fixes in  multiple PRs to ease the review, of course.